libs/cardano-ledger-api/src/Cardano/Ledger/Api/State/Query.hs

```diff
@@ -51,6 +51,12 @@ getDState :: NewEpochState era -> DState era
 getDState = certDState . lsCertState . esLState . nesEs
 
 queryConstitutionHash ::
-  EraGov era => NewEpochState era -> Maybe (SafeHash (EraCrypto era) ConstitutionData)
+  ConwayEraGov era => NewEpochState era -> Anchor (EraCrypto era)
```

Comment:
All queries must be stable, we can't change their return type if that type has a different binary representation. So this function must continue to return a `Maybe (SafeHash`

Another unfortunate restriction imposed on queries is that the must build for all eras, however `ConwayEraGov` prevents that from happening.